### ISSUE TITLE
Build on Device

### ISSUE THREAD

**Issue by apcameron** [starfive-tech/soft_3rdpart#3]
Please provide a Makefile or Cmake Files to build and install these Libraries on the VisionFive 2  as some of us want to be able to recompile/compile these on the VisionFive 2 itself.

**Comment by danboid** [comment]
I would like to second this request. I would like to be able to build the hw video decoding and encoding drivers and ffmpeg locally under Debian please.

**Comment by coderamen666** [comment]
What is stopping you? The StarFive 2 already has a GCC toolchain that runs on device.

**Comment by danboid** [comment]
Have you tried building the waveXXX modules and related programs? The Makefiles etc aren't for Debian Linux.

**Comment by coderamen666** [comment]
What do you mean by that? If you mean it being designed for buildroot then wait for CMake build support.

**Comment by danboid** [comment]
Yes, I mean they are written to be built under buildroot and not Debian.

**Comment by coderamen666** [comment]
There needs to be cmake support
